stylesheets/components/badges.css

```diff
@@ -0,0 +1,91 @@
+.badge-wrapper {
+  background-color: var(--baltic-sea-color);
+  border: 1px solid var(--charcoal-grey);
+  border-radius: var(--border-rd-6);
+  display: flex;
+  min-width: 16rem;
+  justify-content: space-around;
+  padding: 1rem 1.5rem;
+}
+.badge {
+  position: relative;
+}
+.badge-round{
+    position: absolute;
+    z-index: 1;
+    border-radius: 50%;
+}
+/* icons with badge */
+.badge-icon {
+  color: var(--blue-semi-dark);
+  font-size: 2.5rem !important;
+  padding-top: 0.5rem;
+  padding-right: 0.4rem;
+  padding-left: 0.2rem;
+}
+.badge-counter-right {
+  top: 0;
+  right: 0;
+}
+.badge-counter-left {
+  top: 0;
+  left: 0;
+}
+
+.badge-counter { 
+
+  color: var(--white-color);
+  background-color: var(--red-light);
+  padding: 1px 0.4rem;
+}
+/* avatar badge */
+.avatar-badge {
+  width: clamp(2rem, 3.5rem, 3.5rem);
+}
+.avatar-img {
+  border-radius: 50%;
+}
+.badge-status { 
+  padding: 1px 0.4rem;
+}
+.badge-status-dot {
+  min-height: 13.5px;
+  background-color: var(--forest-green);
+  right: 0;
+  bottom: 0.2rem;
+}
+.badge-status-circle {
+  top: 0;
+  height: 100%;
+  width: 100%;
+  border: 2.5px solid var(--blue-semi-dark);
+}
+/* text with badge */
+.badge-text-content {
```

Comment:
Your badge is hiding the text. Try changing top of badge.

Reply:
Agreed! I've done the requested changes. Please check!
<img width="439" alt="image" src="https://user-images.githubusercontent.com/86543154/152143968-6013e454-336b-410a-8ec2-157b9e291a77.png">